ci-operator/jobs/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -105,17 +94,6 @@ periodics:
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-eng-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-control-plane-24nodes
-  reporter_config:
```

Comment:
Hmm, this doesn't seem right.

Let's double check why this is removing slack reporter_config for all jobs.

Also, make sure to add reporter_config for the jobs you added schedules for if you want notifs in the channel.

Reply:
Liquan checked that we don't remove _reporter_config_  in other jobs because the old job type is `always_run: false`. We don't need to add _reporter_config_, but as we change it to `cron`, so we need to manually add _reporter_config_  to the job yaml file.

Reply:
Ok. Whether you need slack notifications for this specific job or not, this PR still shows that reporter_configs are being removed for all the other jobs in the file, which is bad.

It could be some issue with how you generated the file, but we need to revert those changes before merging so we don't lose all our existing reporter_configs.